pkg/justification/processor_test.go

```diff
@@ -284,6 +293,27 @@ func TestCreateToken(t *testing.T) {
 			},
 			wantErr: "unexpected error from validator \"jira\": Cannot connect to validator",
 		},
+		{
+			name: "validator_missing_ui_data",
```

Comment:
I think we should expect error in this case instead of allowing it. If we allow it, the UI rendering will be broken and we can only detect that at run time. @yolocs what do you think?

Reply:
Oh, I see, this is on the processor_test level. The error I was expecting should be from controller level. And seems we are not updating the loading logic yet. Fine to add it in followed-up PR

Reply:
Marked as resolved. Thank you!